mediathread/assetmgr/views.py

```diff
@@ -1046,3 +1046,7 @@ def get(self, request):
         if len(notes) > 0:
             context = {'tags': TagResource().render_related(request, notes)}
         return self.render_to_json_response(context)
+
+
+class BookmarkletMigrationView(TemplateView):
```

Comment:
The upgrade_bookmarklet view is in mediathread.main.views. Probably a good idea to have these two related views in the same place. Agnostic as to whether they stay in main or assetmgr.